transformers/tokenization_utils.py

```diff
@@ -190,6 +190,11 @@ def pad_token_id(self):
         """ Id of the padding token in the vocabulary. Log an error if used while not having been set. """
         return self.convert_tokens_to_ids(self.pad_token)
 
+    @property
+    def pad_token_type_id(self):
+        """ Id of the padding token in the vocabulary. Log an error if used while not having been set. """
```

Comment:
we can remove `Log an error if used while not having been set` I think

Reply:
Very true!